report-ng/app/src/components/classes/classes.ts

```diff
@@ -112,49 +111,47 @@ export class Classes extends AbstractViewModel {
         this._filter();
     }
 
-    private _fillChipList(chipType: ChipType, chipParam?: any){
-        if(Array.isArray(chipParam)){
-            chipParam.forEach(param => {
-                const chip: IChip = {
-                    chipType: chipType,
-                    chipElement: param
+    private _createFilterChips() {
+        this._chips = [];
+
+        for (const type in this.queryParams) {
+            const filter = this.filters.find(filter => filter.type == type);
+
+            if(type != 'config' && type != 'methods'){
+                this.queryParams[type].split("~").forEach(param => {
```

Comment:
Please use standard methods for passing multiple URL query parameters. No custom delimiter magic.

Reply:
I couldn't figure out a better way to modify the queryparams, because the way you proposed earlier is not working (or i could not make it work, here the problem again: https://github.com/telekom/testerra/pull/399#discussion_r1510878084)

What would you propose instead?